source/index.txt

```diff
@@ -6,9 +6,10 @@ Java RS Driver
    :titlesonly:
    :maxdepth: 1
 
-   /installation
-   /get-started
-   /tutorials
+   /installation/
+   /get-started/
+   /tutorials/
+   /reference/
    View the Source <https://github.com/mongodb/mongo-java-driver>
 
 Have a lovely day!
```

Comment:
Question: Do we normally have this? 
If it's insignificant to change, can we remove?

Reply:
This is just the placeholder for the index that I will create! tracked by this [ticket](https://jira.mongodb.org/browse/DOCSP-37875)